core/src/test/java/io/camunda/connector/impl/ConnectorUtilSpec.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.connector.impl;
```

Comment:
Should we move this to the `outbound` package?
```suggestion
package io.camunda.connector.impl.outbound;
```

Reply:
Let's keep it here for now. I'd like to have a general utility for connector related meta-data look-up (potentially re-used by inbound).

We can move any time (it is impl) once the smoke settles around inbound.

Reply:
Yeah, fair enough 👍